pyanaconda/payload/dnfpayload.py

```diff
@@ -893,6 +893,9 @@ def install(self):
                 log.error("Installation failed: %r", e)
                 _failure_limbo()
 
+        if os.path.exists(self._download_location):
+            log.info("Removing existing download location: %s", self._download_location)
```

Comment:
I would suggest "Removing existing package download location:" instead. :)
IIRC there could be more stuff we might download, such as metadata, livepayload, tarpayload, etc.

Reply:
Absolutely. Updated.